vcxagency-node/src/index.js

```diff
@@ -121,10 +123,15 @@ validateAppConfig(appConfig, (err, ok) => {
     appAgent.use('/agency/msg', appAgentMsg)
     appAgent.use('/', appAgentJson)
 
-    appAgent.listen(
-      appConfig.SERVER_PORT,
-      () => logger.info(`Agency is listening on port ${appConfig.SERVER_PORT}!`)
-    )
+    if (appConfig.SERVER_ENABLE_TLS === 'true') {
+      https.createServer({
+        cert: fs.readFileSync('certs/agency.pem'),
+        key: fs.readFileSync('certs/privkey.pem'),
+        ca: fs.readFileSync('certs/ca.pem')
```

Comment:
should be optional, it should be possible to pass only `key + cert` in case of self-signed certs